### PR TITLE
Added prefilledInformation field for paymentRequestWithCardForm() method

### PR DESCRIPTION
**Pull request by itsmepetrov**
- Updated `stripe-ios` to `10.1.0` #49 
- Added `prefilledInformation` field for `paymentRequestWithCardForm()` method #17

### CODE REVIEW

File: ios/TPSStripe/TPSStripeManager.m [hunk 1]
```diff
@@ -463,6 +466,32 @@ - (STPBillingAddressFields)billingType:(NSString*)inputType {
...
+    STPUserInformation *prefilledInformation = [[STPUserInformation alloc] init];
+
+    [prefilledInformation setEmail:inputInformation[@"email"]];
+    [prefilledInformation setPhone:inputInformation[@"phone"]];
+    [prefilledInformation setBillingAddress: [self billingAddress:inputInformation[@"billingAddress"]]];
+
+    return prefilledInformation;
+}
+
+- (STPAddress *)billingAddress:(NSDictionary*)inputAddress {
```
Comment (dmytrozavgorodniy) [inline ios/TPSStripe/TPSStripeManager.m]: I think we should remove `billing` word from this method:

```
- (STPAddress *)address:(NSDictionary*)inputAddress {
     ...
     return address;
}
```

**Comment by dmytrozavgorodniy** [comment]
Looks good to merge

File: example/__tests__/02_test_card_form.js [hunk 2]
```diff
@@ -30,6 +31,15 @@ test('Test if user can use Card Form', async (t) => {
   await driver.keys('4242424242424242 1234 123')
   t.pass('User should be able write card data')
 
+  // Iterate over billing address fields (iOS only)
+  // Verifies that all fields are filled
+  if (platform('ios')) {
+    for (index of new Array(7)) {
```
Comment (isnifer) [inline example/__tests__/02_test_card_form.js]: You've created a global variable 👎

**Comment by cybergrind** [comment]
Uploaded to npm

**Pull request merged**